include/jemalloc/internal/tcache_structs.h

```diff
@@ -54,7 +54,7 @@ struct tcache_slow_s {
 
 struct tcache_s {
 	tcache_slow_t	*tcache_slow;
-	cache_bin_t	bins[SC_NSIZES];
+	cache_bin_t	bins[TCACHE_NBINS_MAX];
```

Comment:
Verified that this goes up to tcache->bins[72], i.e. the 8M size class. So the default config has 32 bins sitting unused (vs right now >200 unused bins).